### DISCUSSION TITLE
Add ability to change published component namespace / location.

### DISCUSSION THREAD

**Discussion by marchershey** [livewire/flux#774] [Feature Requests]
I would like to request the ability to be able to change the location of the published component stubs. Currently, they are located at `resources/views/flux/...` and that is not how my directory structure is set up. 

My request is to have a configuration file that will allow us to set the location of published component stubs, such as ...

```php
return [
    'component_namespace' => 'resources\\views\\components\\flux',
]
```

This is not an important request; it's just my OCD struggling with this one. If it can be added to the todo list, I would greatly appreciate it!

**Comment by joshhanley** [comment]
@marchershey the only issue I can see with that is Flux uses a dedicated syntax `<flux:` instead of `<x-flux:`, but moving them into the components directory would suggest they can be used as Blade components (even though that would probably still work anyway).

What about your Livewire components, where do you currently store them?
